Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamer.cpp

```diff
@@ -660,6 +660,18 @@ static void webKitMediaSrcStreamFlush(Stream* stream, bool isSeekingFlush)
             GST_DEBUG_OBJECT(stream->pad.get(), "StreamingMembers mutex taken, using it to set isFlushing = false.");
             streamingMembers->isFlushing = false;
             streamingMembers->doesNeedSegmentEvent = true;
+
+            if (!webkitGstCheckVersion(1, 21, 3)) {
```

Comment:
1.21.3 is a development version for 1.22.0. Can you check for the latter instead?

Reply:
sure, done